### DISCUSSION TITLE
Minimal Requirements

### DISCUSSION THREAD

**Discussion by FrostDataHuman** [Acly/krita-ai-diffusion#405] [General]
Hello,

Can you provide information about the minimum PC requirements needed to use this program? Many tools like Fooocus, etc., require 4 GB of VRAM.

**Comment by FrostDataHuman** [comment]
I have only gtx 750 ti, so I don't know will it work or not

**Comment by Acly** [comment]
Won't know for sure until you try, but I suspect it's simply too old

**Comment by ghost** [comment]
680m and 8gb ram allocated , doesnt seems to work at all , i wonder what is the minimum requierment for Live .
